src/env.h

```diff
@@ -277,6 +277,23 @@ constexpr size_t kFsStatsBufferLength =
   V(isclosing_string, "isClosing")                                             \
   V(issuer_string, "issuer")                                                   \
   V(issuercert_string, "issuerCertificate")                                    \
+  V(jwk_d_string, "d")                                                         \
+  V(jwk_dp_string, "dp")                                                       \
+  V(jwk_dq_string, "dq")                                                       \
+  V(jwk_dsa_string, "DSA")                                                     \
+  V(jwk_e_string, "e")                                                         \
+  V(jwk_ec_string, "EC")                                                       \
+  V(jwk_g_string, "g")                                                         \
+  V(jwk_k_string, "k")                                                         \
+  V(jwk_p_string, "p")                                                         \
+  V(jwk_q_string, "q")                                                         \
+  V(jwk_qi_string, "qi")                                                       \
+  V(jwk_kty_string, "kty")                                                     \
+  V(jwk_n_string, "n")                                                         \
+  V(jwk_oct_string, "oct")                                                     \
+  V(jwk_rsa_string, "RSA")                                                     \
+  V(jwk_x_string, "x")                                                         \
+  V(jwk_y_string, "y")                                                         \
```

Comment:
The reason why we follow the general `X_string, "X"` format is that it doesn’t matter what these strings are being used for – if some other part of Node.js starts using `n` as a constant string, it’s irrelevant that the original addition was in a JWK context

I would keep following the standard format, even if that breaks up this list over multiple parts of the list here

Reply:
Well... yes and no... there are plenty of examples in that list like `crypto_*_string()`, `dns_*_string()` that don't follow that pattern, and having the prefix makes it much easier to understand the intended context. I'd prefer to keep the prefixes.

Reply:
FWIW, longer term, I'd prefer to move these very specific strings out of the Environment and into the module-specific state binding.

Reply:
We can do that, but keep in mind that de-duplicating string instances is the point here … so a possible long-term solution might be waiting for C++20 to give us fixed strings that can be used as template arguments :P